pkg/plugins/policies/meshhttproute/api/v1alpha1/helpers.go

```diff
@@ -5,7 +5,33 @@ type PolicyDefault struct {
 }
 
 func (x *To) GetDefault() interface{} {
+	var rules []Rule
+
+	for _, rule := range x.Rules {
+		var matches []Match
+
+		for _, match := range rule.Matches {
+			if match.Path == nil {
+				// According to Envoy docs, match must have precisely one of
+				// prefix, path, safe_regex, connect_matcher,
+				// path_separated_prefix, path_match_policy set, so when policy
+				// doesn't specify explicit type of matching, we are assuming
+				// "catch all" path (any path starting with "/").
+				match.Path = &PathMatch{
+					Value: "/",
+					Type:  PathPrefix,
+				}
+			}
```

Comment:
shouldn't this logic be placed where we generate Envoy config? It's sort of Envoy implem details that it always requires path matcher

Reply:
you are right

Reply:
addressed